{"name": "test_delete_contact_unauthorized", "status": "passed", "description": "TC026: Unsuccessful delete contact without auth token", "attachments": [{"name": "log", "source": "ca80cdf1-13e0-4965-b305-aefadbb7bc8d-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "873467ea-af26-468e-96ad-10e153aad22a-attachment.txt", "type": "text/plain"}], "start": 1733918934962, "stop": 1733918935432, "uuid": "cb41ac56-01c8-4077-b61f-0e16da26c338", "historyId": "ddb2a9ee5eb1693b4a2ec24f0c45143c", "testCaseId": "ddb2a9ee5eb1693b4a2ec24f0c45143c", "fullName": "API.test_api#test_delete_contact_unauthorized", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "API"}, {"name": "tag", "value": "critical"}, {"name": "parentSuite", "value": "API"}, {"name": "suite", "value": "test_api"}, {"name": "host", "value": "5cbf70c9ed4a"}, {"name": "thread", "value": "1-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "API.test_api"}]}